src/Raven.Client/Documents/Operations/ETL/EtlProcessState.cs

```diff
@@ -23,6 +23,8 @@ public EtlProcessState()
 
         public string NodeTag { get; set; }
 
+        public HashSet<string> SkippedTimeSeriesDocs { get; set; }
```

Comment:
No, when a node skips time-series while ETL because its document didn't replicate yes and then the ETL pass to another node the new node will skip this time-series because it was "processed" (skipped) by the first node so the time-series will not be marked as skipped by the new node. So when the new node will get to ETL the document it will see the document time-series was skipped by the first node and so will load the time-series with it.

Reply:
OK. Still a comment explaining why is this collection used for would be very nice.